api/src/json_utils.rs

```diff
@@ -34,3 +34,47 @@ pub fn access_state_error<E: fmt::Debug>(e: E) -> JsonRpcError {
 pub fn transaction_error<E: fmt::Debug>(e: E) -> JsonRpcError {
     JsonRpcError::without_data(3, format!("Execution reverted: {e:?}"))
 }
+
+pub fn parse_params_0(request: serde_json::Value) -> Result<(), JsonRpcError> {
```

Comment:
I feel it might call for a little `macro_rules!` here. Though AFAIK @bulenttastan is usually against it, but shouldn't hurt readability / maintainability too much. Also otherwise I would advocate for `parse_zero...`, `parse_one...` etc naming, but that's just a nit.